src/components/tables/RowActions/Shared/Progress.tsx

```diff
@@ -86,22 +93,24 @@ function SharedProgress({
                 ) : null}
             </Box>
 
-            <Box sx={wrapperStyling}>
-                {state !== ProgressStates.RUNNING && logToken !== null ? (
-                    renderLogs ? (
-                        renderLogs(logToken)
-                    ) : (
-                        <ErrorLogs
-                            logToken={logToken}
-                            height={150}
-                            logProps={{
-                                disableIntervalFetching: true,
-                                fetchAll: true,
-                            }}
-                        />
-                    )
-                ) : null}
-            </Box>
+            {renderLogs && logToken !== null ? (
+                <Box sx={wrapperStyling}>
+                    {state !== ProgressStates.RUNNING ? (
+                        typeof renderLogs === 'function' ? (
+                            renderLogs(logToken)
+                        ) : (
+                            <ErrorLogs
+                                logToken={logToken}
+                                height={150}
+                                logProps={{
+                                    disableIntervalFetching: true,
+                                    fetchAll: true,
+                                }}
+                            />
+                        )
+                    ) : null}
+                </Box>
+            ) : null}
```

Comment:
If there is nothing to render we don't need the box. I did try making this a collapse and having this section animate in but it looked weird when multiple rows were showing errors.